### PR TITLE
Validate sqf - python 3 compat

### PR DESCRIPTION
**Pull request by PabstMirror**
Should be python 2x compatible

### CODE REVIEW

File: tools/sqf_validator.py [hunk 1]
```diff
@@ -84,13 +81,13 @@ def popClosing():
                         checkIfNextIsClosingBlock = False
 
         if brackets_list.count('[') != brackets_list.count(']'):
-            print "A possible missing [ or ] in file " + filepath + "[ = " + str(brackets_list.count('[')) + " ] =" + str(brackets_list.count(']'))
+            print("A possible missing square bracket [ or ] in file {0} [ = {1} ] = {2}".format(filepath,brackets_list.count('['),brackets_list.count(']')))
             bad_count_file += 1
         if brackets_list.count('(') != brackets_list.count(')'):
-            print "A possible missing ( or ) in file " + filepath + "( = " + str(brackets_list.count('(')) + " ) =" + str(brackets_list.count(')'))
+            print("A possible missing round bracket ( or ) in file {0} ( = {1} ) = {2}".format(filepath,brackets_list.count('('),brackets_list.count(')')))
```
Comment (bux578) [inline tools/sqf_validator.py]: parenthesis?

File: tools/sqf_validator.py [hunk 2]
```diff
@@ -84,13 +81,13 @@ def popClosing():
...
             bad_count_file += 1
         if brackets_list.count('{') != brackets_list.count('}'):
-            print "A possible missing { or } in file " + filepath + "{ = " + str(brackets_list.count('{')) + " } =" + str(brackets_list.count('}'))
+            print("A possible missing curly brace {{ or }} in file {0} {{ = {1} }} = {2}".format(filepath,brackets_list.count('{'),brackets_list.count('}')))
```
Comment (bux578) [inline tools/sqf_validator.py]: just `brace`?

File: tools/sqf_validator.py [hunk 3]
```diff
@@ -84,13 +81,13 @@ def popClosing():
                         checkIfNextIsClosingBlock = False
 
         if brackets_list.count('[') != brackets_list.count(']'):
-            print "A possible missing [ or ] in file " + filepath + "[ = " + str(brackets_list.count('[')) + " ] =" + str(brackets_list.count(']'))
+            print("A possible missing square bracket [ or ] in file {0} [ = {1} ] = {2}".format(filepath,brackets_list.count('['),brackets_list.count(']')))
```
Comment (bux578) [inline tools/sqf_validator.py]: Just `bracket`?

File: tools/sqf_validator.py (hunk 2 above)
Comment (jonpas) [inline tools/sqf_validator.py]: It's either `curly bracket` or `brace`.

**Pull request merged**